src/platform/forms-system/src/js/patterns/array-builder/README.md

```diff
@@ -444,7 +445,27 @@ const options = {
 | `updated=nounSingular_0` | Used after completing an edit flow. |
 | `removedAllWarn=true` | Used after removing all items. Will show a warning message if the item is required. |
 
-## Advanced routing example with `helpers`
+## Advanced routing
+Use `depends` for conditional pages
+```js
+...arrayBuilderPages(employersOptions, pageBuilder => ({
+  multiPageBuilderSummary: pageBuilder.summaryPage({
+    title: 'Array with multiple page builder summary',
+    path: 'array-multiple-page-builder-summary',
+    uiSchema: employersSummaryPage.uiSchema,
+    schema: employersSummaryPage.schema,
+  }),
+  multiPageBuilderStepOne: pageBuilder.itemPage({
+    title: 'Employer name and address',
+    path: 'array-multiple-page-builder/:index/name-and-address',
+    uiSchema: employersPageNameAndAddressPage.uiSchema,
+    schema: employersPageNameAndAddressPage.schema,
+    depends: (formData, index) => formData.employers?.[index]?.type === 'Military',
```

Comment:
If `formData.employers` is an array in this example, then shouldn't the syntax be this to access the type property:
```suggestion
    depends: (formData, index) => formData.employers[index]?.type === 'Military',
```